src/System.Memory/tests/ReadOnlySpan/IndexOfAny.char.cs

```diff
@@ -321,11 +321,276 @@ public static void MakeSureNoChecksGoOutOfRangeThree_Char()
             }
         }
 
+        [Fact]
+        public static void ZeroLengthIndexOfFour_Char()
+        {
+            ReadOnlySpan<char> sp = new ReadOnlySpan<char>(Array.Empty<char>());
+            ReadOnlySpan<char> values = new char[] { (char)0, (char)0, (char)0, (char)0 };
+            int idx = sp.IndexOfAny<char>(values);
+            Assert.Equal(-1, idx);
+        }
+
+        [Fact]
+        public static void DefaultFilledIndexOfFour_Char()
+        {
+            Random rnd = new Random(42);
+
+            for (int length = 0; length < byte.MaxValue; length++)
+            {
+                char[] a = new char[length];
+                ReadOnlySpan<char> span = new ReadOnlySpan<char>(a);
+
+                char[] targets = { default, (char)99, (char)98, (char)97 };
+
+                for (int i = 0; i < length; i++)
```

Comment:
So we're doing this test ~32K times?  And the first time through the outer loop (length == 0), we're not testing anything?

Reply:
Changed outer loop to == 1

Reply:
Now varying the start position from 0 - Vector<short>.Count

Ignoring the actual values; it probably only needs to loop `Vector<short>.Count` * 4 to hit all the alignment, vector and trailing loops

Which would be 128 on AVX512; is it worth shorting it?